outagedetector/outage_detector.py

```diff
@@ -33,10 +33,14 @@ def check_power_and_internet(run, notification):
         just_booted = True
     elif run == "scheduled":
         just_booted = False
-    if notification == "notification":
+    if notification == "notification" or "ifttt":
```

Comment:
This if statement is broken, needs to be
`if notification == "notification" or notification == "ifttt":`